apt/private/deb_translate_lock.bzl

```diff
@@ -44,22 +44,30 @@ load("@rules_distroless//apt:defs.bzl", "dpkg_status")
 
 exports_files(['packages.bzl'])
 
+# Map Debian architectures to platform CPUs.
+#
+# For more info on Debian architectures, see:
+#     * https://wiki.debian.org/SupportedArchitectures
+#     * https://wiki.debian.org/ArchitectureSpecificsMemo
+#     * https://www.debian.org/releases/stable/amd64/ch02s01.en.html#idm186
+#
+# For more info on Bazel's platforms CPUs see:
+#     * https://github.com/bazelbuild/platforms/blob/main/cpu/BUILD
 _ARCHITECTURE_MAP = {{
     "amd64": "x86_64",
-    "arm64": "aarch64",
```

Comment:
this is a great catch, should we add all possible combination even if they are identical just for completeness? 
eg "arm64": "arm64".

Reply:
Ah, I see! That's probably better than fall-though, set all the architectures that should be there and fail if not in the dict.

OK, will add them to the map and will re-word the commit!